code/SQLite3Connector.php

```diff
@@ -170,7 +170,7 @@ public function quoteString($value)
 
     public function escapeString($value)
     {
-        return $this->dbConn->escapeString($value);
+        return $this->dbConn->escapeString((string)$value);
```

Comment:
```suggestion
        return $this->dbConn->escapeString($value ?? '');
```
Casting the value to a string might hide situations where some other non-string value is passed in by mistake. In this case we're explicitly only trying to avoid issues if the value is null which has, until now, been perfectly valid.